mp/game/momentum/resource/ui/SettingsPanel_OnlineSettings.res

```diff
@@ -79,13 +79,44 @@
 		"Font"		"DefaultVerySmall"
 	}
     
+	"ToggleEntityPanels"
```

Comment:
This also needs to be declared in the code. See how the other elements are declared in the OnlineSettings class.

Reply:
In doing this I found that the only checkbox declared in the code (`HudSettingsPage`) is the one for `mom_ghost_online_alpha_override_enable`. Should I add the others as well?

Reply:
Yes.

Reply:
Done.